_includes/help/nav_page_bottom.html

```diff
@@ -0,0 +1,33 @@
+<div class="sm-hide bg-blue-lightest">
+  <nav class="bg-blue-lightest px0">
+    <ul class="js-accordion p0 m0 list-reset">
+    {% for subpage in site.help_pages %}
+      <li>
+        {% assign subpage_slug = subpage | slugify %}
+        {% assign related_questions = site.help | where_exp: "item", "item.url contains subpage_slug" | sort: 'order' %}
+        {% capture is_initially_expanded %}{% if page.url contains subpage_slug %}true{% else %}false{% endif %}{% endcapture %}
+        {% capture is_initially_hidden %}{% if page.url contains subpage_slug %}false{% else %}true{% endif %}{% endcapture %}
+        <button class="btn-transparent border-bottom {% if forloop.first %}border-top{% endif %} flex flex-center w-100pc left-align p2" aria-expanded="{{ is_initially_expanded }}">
+          <span class="w-100pc">{{ subpage }}</span>
+          <span class="svg-wrapper svg-gray">{% include svg/carat-down.svg %}</span>
```

Comment:
Would it be possible to flip the caret the other direction when the menu is expanded?